app/views/users/show.html.erb

```diff
@@ -1,2 +1,63 @@
-<h1>Users#show</h1>
-<p>Here is the details of a user that was created</p>
+<section class='container'> 
+    <div class="card d-flex">
+        <div class="card-image">
+            <%= image_tag('posts/ball.jpg', width:'100%', height:'100%') %>
+        </div>
+        <div class="card-body">
+            <h1 class="card-title"> <%= @user_info['username'] %> </h1>
+
+            <p class="card-text">Number of posts: <%= @user_info['number_of_posts'] %> </p>
+        </div>
+    </div>
+
+    <div class='bio-wrapper'>
+        <div class='bio-div'>
+          <h4> Bio </h4>
+          <p >Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid iure possimus aut, soluta tenetur reiciendis placeat eos sunt totam consequuntur labore quas atque ipsa rem repellendus dolore suscipit! Laudantium, voluptas.Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid iure possimus aut, soluta tenetur reiciendis placeat eos sunt totam consequuntur labore quas atque ipsa rem repellendus dolore suscipit! Laudantium, voluptas. </p>
+        </div>
+    </div>
+
+    <%= link_to user_posts_path(@user_info['id']) do %>
+        <div class='post-wrapper'>
+            <div class='post-div'>
+                <h4>Post #1 </h4>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid iure possimus aut, soluta tenetur reiciendis placeat eos sunt totam consequuntur labore quas atque ipsa rem repellendus...
+                </p>
+                <p class='comments-likes'>
+                    <small>Comments:x, Likes:y </small>
+                </p>
+            </div>
+        </div>
+    <% end %>
+  
+    <%= link_to user_posts_path(@user_info['id']) do %>
+        <div class='post-wrapper'>
+            <div class='post-div'>
+                <h4>Post #1 </h4>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid iure possimus aut, soluta tenetur reiciendis placeat eos sunt totam consequuntur labore quas atque ipsa rem repellendus...
+                </p>
+                <p class='comments-likes'>
+                    <small>Comments:x, Likes:y </small>
+                </p>
+            </div>
+        </div>
+    <% end %>
+  
+       <%= link_to user_posts_path(@user_info['id']) do %>
+        <div class='post-wrapper'>
+            <div class='post-div'>
+                <h4>Post #1 </h4>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid iure possimus aut, soluta tenetur reiciendis placeat eos sunt totam consequuntur labore quas atque ipsa rem repellendus...
+                </p>
+                <p class='comments-likes'>
+                    <small>Comments:x, Likes:y </small>
+                </p>
+            </div>
+        </div>
+    <% end %>
+  
+    <div> <button class='see-all-posts'> See all posts</button> </div>
```

Comment:
- [ ] It would be better if we add the path to the button so that users can interact with the button